coq-topology.opam

```
@@ -16,7 +16,7 @@ and results of general topology in Coq.
 build: [make "-j%{jobs}%" ]
 install: [make "install"]
 depends: [
-  "coq" {(>= "8.11" & < "8.14~") | (= "dev")}
+  "coq" {(>= "8.10" & < "8.14~") | (= "dev")}
   "coq-zorns-lemma" {= "dev"}
```

This will be equivalent in this repository (the current version is indeed "dev") but will make sense when exporting to the opam-coq-archive since released versions of topology will depend on the same version of zorns-lemma:

```suggestion
  "coq-zorns-lemma" {= version}
```